### PR TITLE
Update publication_credit.md

### PR DESCRIPTION
**Pull request by sateeshperi**
add doi and publications links

### CODE REVIEW

File: sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md [hunk 1]
```diff
@@ -4,16 +4,30 @@ subtitle: Recommended recognition for nf-core pipeline publications
...
 ---
 
-Pipeline publications should acknowledge the nf-core community (e.g. in an academic journal).
+## Acknowledging nf-core in Academic Publications
 
-**At a minimum**, the nf-core community should be thanked in the acknowledgment section. A good example is from the [nf-core/clipseq](https://doi.org/10.12688/wellcomeopenres.19453.1)) preprint:
+If your work has utilized nf-core pipelines or elements of the framework, please ensure that you cite the following publication, which describes the community and framework in detail:
 
-> We would also like to thank the nf-core team, in particular Harshil Patel, Phil Ewels and Maxime Garcia, for welcoming us to the nf-core community and their assistance in reviewing and releasing the pipeline. We would also like to thank the nf-core community for developing the nf-core infrastructure and resources for Nextflow pipelines. A full list of nf-core community members is available at https://nf-co.re/community.
+> Ewels, P.A., Peltzer, A., Fillinger, S. et al. The nf-core framework for community-curated bioinformatics pipelines. Nat Biotechnol 38, 276–278 (2020). https://doi.org/10.1038/s41587-020-0439-x
```
Comment (jfy133) [inline sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:11]: What about hte more recent preprint? Or we wait until it's published published?

File: sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md [hunk 2]
```diff
@@ -4,16 +4,30 @@ subtitle: Recommended recognition for nf-core pipeline publications
...
 
-Other examples: ([nf-core/eager](https://peerj.com/articles/10947/#acknowledgements)).
+### Minimum Requirement: Acknowledgments Section
 
-**Optionally**, the nf-core community can be included as a consortium co-author of the publication ([example](https://doi.org/10.3390/ijms232314512)). This route is a good idea when a pipeline has used extensive existing nf-core pipeline infrastructure (e.g. modules) that were written by other members of the community not directly involved in the pipeline itself.
+At a minimum, the **nf-core** community should be acknowledged in the **Acknowledgments** section of your paper. Here’s an example from the [nf-core/clipseq preprint](https://doi.org/10.12688/wellcomeopenres.19453.1):
 
-If any members of the nf-core community have provided significant input to the creation of the pipeline, please consider adding them as coauthors on the paper directly.
```
Comment (jfy133) [inline sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:17]: I would still have a separate sentence saying you should thank specific people too (and consider the co-author ship) here, I think many won't look at the next two sections if indicated as 'optional'

Comment (jfy133) [reply sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:17]: OR ust remove optional from the consortoum thing

File: sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md [hunk 3]
```diff
@@ -4,16 +4,30 @@ subtitle: Recommended recognition for nf-core pipeline publications
 weight: 123
 ---
 
-Pipeline publications should acknowledge the nf-core community (e.g. in an academic journal).
+## Acknowledging nf-core in Academic Publications
 
-**At a minimum**, the nf-core community should be thanked in the acknowledgment section. A good example is from the [nf-core/clipseq](https://doi.org/10.12688/wellcomeopenres.19453.1)) preprint:
+If your work has utilized nf-core pipelines or elements of the framework, please ensure that you cite the following publication, which describes the community and framework in detail:
```
Comment (ewels) [inline sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:9]: I think this changes the scope of this page significantly.

The previous text read _Pipeline publications_ - specifically publications about nf-core pipelines.

Acknowledgements, co-authorship and the rest is excessive if someone is just running an nf-core pipeline for their analysis.

File: sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md [hunk 4]
```diff
@@ -4,16 +4,30 @@ subtitle: Recommended recognition for nf-core pipeline publications
...
+
+If your work has made extensive use of **nf-core** pipeline infrastructure (e.g., multiple modules written by other community members), consider including the **nf-core community** as a **consortium co-author**. This approach is ideal when a significant portion of the pipeline was developed by contributors who were not directly involved in your specific project. See an example of this in [this publication](https://doi.org/10.3390/ijms232314512).
+
+### Co-authorship for Significant Contributions
+
+If individual members of the **nf-core** community have made significant contributions to your pipeline’s development, please consider inviting them as co-authors on the paper. A good rule of thumb is to include those who have provided substantial input beyond the standard community support.
+
+### Need Guidance?
+
+If you’re unsure how to acknowledge **nf-core** in your publication, feel free to reach out to the **core team** for advice. They’ll be happy to guide you on the best practices for recognition.
```
Comment (ewels) [inline sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:33]: Would be good to say how to do this. I usually write "ask on slack" as that's more actionable. Could even be "Ping `@core-team` on Slack".

File: sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md (hunk 3 above)
Comment (sateeshperi) [reply sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:9]: hmmm not sure i understand. co-authorship and others were also mentioned previously...I just made them into sub-sections. are you saying to just to make them less obvious or remove them all together ?

Comment (jfy133) [reply sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:9]: I think what Phil means is this page is aimed at papers publishing an nf- core pipeline. So announcing to the world a particular pipeline exists. 

By rephrasing to 'utilising' a pipeline, it now is saying if you use any nf-core pipeline to analyse data, you also have to cite the main nf-core paper and acknowledge the community, which is asking a bit much .

At least this is how I'm reading it and I think I'm getting the same impression as Phil

Comment (sateeshperi) [reply sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:9]: ah i c u now

**Comment by sateeshperi** [comment]
reverted back mostly and added DOI and publications links. how does it sound now ? @ewels

File: sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md [hunk 5]
```diff
@@ -4,30 +4,20 @@ subtitle: Recommended recognition for nf-core pipeline publications
 weight: 123
 ---
 
-## Acknowledging nf-core in Academic Publications
-
-If your work has utilized nf-core pipelines or elements of the framework, please ensure that you cite the following publication, which describes the community and framework in detail:
+Pipeline publications should acknowledge the nf-core community (e.g. in an academic journal). Please cite the following publication, which describes the community and framework in detail:
```
Comment (jfy133) [inline sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md:7]:
```suggestion
Pipeline publications (or papers within which pipelines are described for the first time) should acknowledge the nf-core community (e.g. in an academic journal). Please cite the following publication, which describes the community and framework in detail:
```

**Review by jfy133 — APPROVED** [review]

File: sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md (hunk 3 above)
Comment (ewels) [reply sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md]: Yup, that ☝🏻

Comment (sateeshperi) [reply sites/docs/src/content/docs/guidelines/pipelines/recommendations/publication_credit.md]: ya reverted back most of that language to keep it minmal

**Review by ewels — APPROVED** [review]
LGTM, thanks!

**Pull request merged**